### DISCUSSION TITLE
Node v14 prebuild binaries for windows

### DISCUSSION THREAD

**Discussion by viceice** [uhop/node-re2#130] [Q&A]
Why the prebuild binaries for windows for node v14 are skipped?

https://github.com/uhop/node-re2/blob/f922bbe576060864b13741043e5696b21338fbc7/.github/workflows/build.yml#L31-L37

**Comment by uhop** [comment]
It fails building the C++ part and I have no Windows to debug it. I decided to skip that particular version.

**Reply by viceice** [reply]
ok, that issue seems somehow to be hidden on yarn install. so the native module was simply missing 😕 

we now switched our windows tests to node v16, so no need to fix that for us (renovate).
